sql/item_mods.sql

```diff
@@ -5954,9 +5958,9 @@ INSERT INTO `item_mods` VALUES (11925,26,9);
 INSERT INTO `item_mods` VALUES (11926,1,40);
 INSERT INTO `item_mods` VALUES (11926,10,25);
 INSERT INTO `item_mods` VALUES (11926,426,35);
-INSERT INTO `item_mods` VALUES (11927,1,40);
-INSERT INTO `item_mods` VALUES (11927,385,250);
-INSERT INTO `item_mods` VALUES (11927,831,-50);
+INSERT INTO `item_mods` VALUES (11927,1,40); -- def
+INSERT INTO `item_mods` VALUES (11927,385,250); -- shield bash
+INSERT INTO `item_mods` VALUES (11927,163,-50); -- Aegis (lv99 no afterglow) -50% Magic Damage Taken
```

Comment:
@Hozu I added 831 back into Aegis but found it also assigned to MOD_Quick_Draw in status.lua